ui/job-view/pushes/Revision.jsx

```diff
@@ -59,6 +60,12 @@ export class Revision extends React.PureComponent {
       <li className="clearfix">
         <span className="revision" data-tags={this.tags}>
           <span className="revision-holder">
+            <CopyToClipboard text={commitRevision}>
+              {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+              <a title="Copy full hash">
```

Comment:
I'd get rid of the anchor tag and give the `FontAwesomeIcon` or a span a className='pointer' (that class is in the treeherder-global stylesheet). Also, giving the FontAwesomeIcon a title attribute will provide an `aria-labelledby` tag: https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility#svg-semantic

Reply:
Thanks a lot for your suggestions!